OpenEdXMobile/res/layout/fragment_course_unit_grade.xml

```diff
@@ -5,71 +5,85 @@
+                    android:orientation="vertical"
+                    android:padding="@dimen/edx_margin">
 
-            <androidx.appcompat.widget.AppCompatButton
-                android:id="@+id/btn_learn_more"
-                style="@style/edX.Widget.PrimaryActionButton"
-                android:layout_marginEnd="@dimen/widget_margin"
-                android:includeFontPadding="false"
-                android:padding="@dimen/container_padding"
-                android:text="@string/course_value_prop_button"
-                app:layout_constraintEnd_toEndOf="parent"
-                app:layout_constraintTop_toBottomOf="@+id/ll_graded_content_description" />
-        </androidx.constraintlayout.widget.ConstraintLayout>
+                    <TextView
+                        android:id="@+id/ll_graded_content_title"
+                        android:layout_width="wrap_content"
+                        android:layout_height="wrap_content"
+                        android:drawablePadding="@dimen/widget_margin"
+                        android:fontFamily="?attr/fontSemiBold"
+                        android:gravity="center"
+                        android:text="@string/course_modal_graded_assignment_locked"
+                        android:textColor="@color/neutralBlackT"
+                        android:textSize="@dimen/edx_small"
+                        app:drawableStartCompat="@drawable/ic_lock"
+                        app:drawableTint="@color/neutralBlackT" />
+
+                    <TextView
+                        android:id="@+id/ll_graded_content_description"
+                        android:layout_width="wrap_content"
+                        android:layout_height="wrap_content"
+                        android:layout_marginTop="@dimen/edx_margin"
+                        android:lineSpacingMultiplier="@dimen/course_date_desc_line_spacing"
+                        android:text="@string/course_modal_graded_assignment_locked_description"
+                        android:textColor="@color/neutralXXDark"
+                        android:textSize="@dimen/edx_medium" />
+
+                    <androidx.appcompat.widget.AppCompatButton
+                        android:id="@+id/toggle_show"
+                        android:layout_width="wrap_content"
+                        android:layout_height="wrap_content"
+                        android:layout_marginTop="@dimen/widget_margin_x_double"
+                        android:layout_marginBottom="@dimen/widget_margin_x_double"
+                        android:text="@string/course_modal_graded_assignment_show_more"
+                        android:textColor="@color/neutralXXDark"
+                        android:textSize="@dimen/edx_medium" />
+
+                    <include
+                        android:id="@+id/layout_upgrade_feature"
+                        layout="@layout/layout_upgrade_features"
+                        android:visibility="gone" />
```

Comment:
IMO, we have to place the `upgrade_feature` within the `scrollview` cuz, in the devices have a small height size, didn't shows the `upgrade_button` & some list items.
<img src="https://user-images.githubusercontent.com/43750646/128162436-d1847421-b47a-4e29-888f-41d72fd24c30.png" height="512"/>

Reply:
@miankhalid thoughts?

Reply:
I agree, this should be in a scrollview.

Reply:
ScrollView implemented